packages/cli/src/commands/create/dependencies.ts

```diff
@@ -76,9 +76,11 @@ export async function installDependencies(cwd: string, flags: CreateOptions) {
 async function getPackageManager() {
   const installedPackageManagers = await getInstalledPackageManagers();
 
+  const initialValue = process.env.npm_config_user_agent?.split('/')[0] ?? 'npm';
```

Comment:
I think it would be good if we double-checked to make sure it's part of the installed package managers?

```ts
  let initialValue = process.env.npm_config_user_agent?.split('/')[0] as PackageManager | undefined;

  if (!installedPackageManagers.includes(initialValue)) {
    initialValue = 'npm';
  }
```

Reply:
I agree, either that or we assume that the value provided will always work (given it succeeded already) and we make sure that `installPackageManagers` includes it.

Reply:
@SamVerschueren @Nemikolh Thank you for your reviews! I've updated the code to match Sam's suggestion!